TitanRobotics2023/src/main/java/frc/robot/drivecontrol

```diff
@@ -0,0 +1,30 @@
+package frc.robot.actors;
+
+import frc.robot.data.PortMap;
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj.motorcontrol.PWMVictorSPX;
+import edu.wpi.first.wpilibj.drive.MecanumDrive;
+
+public class driveControl
+
+{
+
+private final MotorController motor_frontLeft;
+private final MotorController motor_frontRight;
+private final MotorController motor_backLeft;
+private final MotorController motor_bakcRight;
+
+  public DriveControl()
+  {
+   motor_frontLeft = new PWMVictorSPX(PortMap.FRONTLEFT.portNumber);
```

Comment:
Can you put mortor_frontLeft the same spacing. So that I don't have a heart attack every time.